tests/kernel/workq/work_queue_api/src/main.c

```diff
@@ -135,19 +164,46 @@ void test_process_work_items_fifo(void)
 }
 
 /**
- * @brief Test kernel support scheduling work item that is to be processed
- * after user defined period of time
+ * @brief Test submitting a delayed work item to a workqueue
  * @details
- * - For that test is using semaphore sync_sema, with initial count 0.
- * - In that test we measure actual spent time and compare it with time
+ * Test Objective:
+ * - Test kernel support scheduling work item that is to be processed
+ * after user defined period of time.
+ *
+ * Test techniques:
+ * - Dynamic analysis and testing
+ * - Interface testing
+ *
+ * Prerequisite Conditions:
+ * - N/A
+ *
+ * Input Specifications:
+ * - N/A
+ *
+ * Test Procedure:
+ * -# For that test is using semaphore sync_sema, with initial count 0.
+ * -# In that test we measure actual spent time and compare it with time
  * which was measured by function k_delayed_work_remaining_get().
- * - Using system clocks we measure actual spent time
+ * -# Using system clocks we measure actual spent time
  * in the period between delayed work submitted and delayed work
  * executed.
- * - To know that delayed work was executed, we use semaphore.
- * - Right after semaphore was given from handler function, we stop
+ * -# To know that delayed work was executed, we use semaphore.
+ * -# Right after semaphore was given from handler function, we stop
  * measuring actual time.
- * - Then compare results.
+ * -# Then compare results.
+ *
+ * Expected Test Result:
+ * - The measured time results are in a very small range.
```

Comment:
Here, it's better to specify that range.

Reply:
@yingmingx It seems like the "check point" was not been pointed out in your test procedure, it's a little ambiguous here. 
You could declare it like this:  "Then check if the X-time is within Y and Z".    or  something likes that.